app/src/main/java/com/github/se/eventradar/model/event/EventCategory.kt

```diff
@@ -12,6 +12,12 @@ enum class EventCategory(val displayName: String) {
   SOCIAL("Social"),
   PARTY("Party");
 
+  companion object {
+    fun fromDisplayName(displayName: String): EventCategory? {
```

Comment:
see above comment, but i don't believe this will work with the current Firebase schema.